documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -439,3 +439,82 @@ characteristicDefinitionsUrl.ifPresent( url -> {
 } );
 ----
 
+[[mapping-aas]]
+== Mapping Aspect Models to Asset Administration Shell (AAS) Submodel Templates
+The Asset Administration Shell (AAS) and its https://www.plattform-i40.de/IP/Redaktion/EN/Standardartikel/specification-administrationshell.html[information model] is a widely recognized standard developed by the https://industrialdigitaltwin.org[Industrial Digital Twin Association (IDTA)] to express and handle Digital Twins. Central element of the AAS is
+the concept of Submodels, which describe certain aspects of a Digital Twin.
+
+The BAMM Aspect Meta Model allows to specify aspects of a digital twin and its semantics.
+The AAS Generator module provides mapping implementations to derive AAS Submodels from BAMM Aspect models
+and by that allows to on the one hand integrate BAMM models in AAS environments and on the other hand allow
+AAS Submodels to be described with rich semantics, as it is possible with BAMM.
+
+[[details-mapping-aas]]
+== Details to the Mapping Concept
+In the following section, the mapping rules applied by the generator are explained.
+The rules apply to https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/index.html[BAMM v1.0.0] and
+https://www.plattform-i40.de/IP/Redaktion/EN/Downloads/Publikation/Details_of_the_Asset_Administration_Shell_Part1_V3.html[AAS Specification Part 1 V3.0RC01].
+
+[cols="1,1,2"]
+|===
+| BAMM  | AAS  | Comment
+
+| **bamm:Aspect**  |  aas:Submodel with kind=Template  | Empty Asset and AssetAdministrationShell entries are added to the output file
+
+| bamm:name   | aas:Submodel.idShort  |
+
+| bamm:preferredName   | aas:Submodel.displayName  |
+
+| bamm:description   | aas:Submodel.description  |
+
+| bamm:property   | see **bamm:Property**  |
+
+| bamm:operation   | see **bamm:Operation**   |
+
+| bamm:Aspect.urn  | aas:Submodel.semanticId  |
+
+| **Property**  | aas:Property <br> aas:SubmodelElementCollection   | The AAS type is derived from the type of the BAMM Characteristic specifying the BAMM property. Depending on the type it is decided what the resulting AAS element will be. In case of an Entity it will result in a SubmodelElementCollection. It will also be a SubmodelElementCollection if the BAMM Characteristic is of a Collection type (see the https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/characteristics.html[Characteristics taxonomy]). In all other cases an aas:Property will be generated
```

Comment:
Literal `<br>` will not work in Antora. To force a line break inside the cell, allow Asciidoc-Markup inside the table cell by marking the column `a`, i.e. change the line above the table to `[cols="1,1a,2"]`, then in the cell, instead of `<br>`, add a single empty line. Repeat for other `<br>` occurences in the table.

Also please check whether it is correct that there is no comma between the referenced elements in the second colum, while other cells below mention two elements but separate them using a comma.